server/admin/admin_server.go

```diff
@@ -42,6 +42,11 @@ func New(registry extension.Registry) *Server {
 	return &Server{registry}
 }
 
+// IsHealthy returns nil if the server is healthy, error otherwise.
+func (s *Server) IsHealthy() error {
+	return s.registry.AdminStorage.CheckDatabaseAccessible(context.Background())
```

Comment:
I realize the other IsHealthy methods don't take a ctx either, but maybe we should break the consistency here and take a ctx as parameter. WDYT?